package.json

```diff
@@ -23,8 +23,11 @@
     "postcss": "8.4.24",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-markdown": "^9.0.1",
     "react-scroll": "^1.8.9",
     "react-type-animation": "^3.2.0",
+    "rehype-raw": "^7.0.0",
+    "remark-gfm": "^4.0.0",
```

Comment:
Do we really need these dependencies ? I saw they have a few starts and most important the last commit is dated September 2023.

Reply:
What approach would you suggest then? These packages seem to do what they are intended to do...

Reply:
adding new dependencies is always something to really think about because it could create really a lot of problems and work to do in the long term (security and maintenance), in particular in the frontend when there is a proliferation of dependencies.

Having said that, even if the last commits are from the last year, they seem very well integrated packages in the ecosystem and some more power to the markdown of IntelOwl's blog does make sense

Reply:
Ok, let's use them

Reply:
great! Let's merge it when I'm done with the blog?